docs/OPEN_SOURCE_LICENSES.txt

```diff
@@ -710,9 +710,8 @@
 │  ├─ url: https://github.com/jonschlinkert
 ├─ bcgov-digital-marketplace@1.0.0
 │  ├─ licenses: Apache-2.0
-│  ├─ repository: 
-│  ├─ publisher: Real Folk Inc.
-│  └─ noticeFile: /home/dd/src/bcgov/digital_marketplace/NOTICE.txt
+│  ├─ repository:
+│  ├─ publisher: Procurement Services Branch within the Government of British Columbia's Ministry of Citizen's Services
```

Comment:
@bdolor In light of your comment on the `LICENSE.txt` should I return this to the original state by the same logic?
I'm far from an expert on licenses, happy to ask Steve too

Reply:
Returning to the original state is consistent with language in the Apache license regarding the work of authorship.